src/main/java/digital/slovensko/autogram/core/SigningJob.java

```diff
@@ -221,6 +225,6 @@ private static SigningParameters getParametersForFile(FileDocument document, boo
     }
 
     public boolean shouldCheckPDFCompliance() {
-        return parameters.getCheckPDFACompliance() && isPDF(document.getMimeType());
+        return parameters.getCheckPDFACompliance() && isPDF(document.getMimeType()) && !document.isProtected();
```

Comment:
prečo?

Reply:
org.verapdf hádže chybu, že enkryptované pdfka nie sú podporované. Je to napísané aj na [wikipedií](https://en.wikipedia.org/wiki/PDF/A#:~:text=PDF/A%20differs%20from%20PDF%20by%20prohibiting%20features%20unsuitable%20for%20long%2Dterm%20archiving%2C%20such%20as%20font%20linking%20(as%20opposed%20to%20font%20embedding)%20and%20encryption.). Toto upravím, keďže ako si písal vyššie, s PDF/A môže existovať s master password ale nie s open document password - teda enkrypciou.

Reply:
okay. Pridaj tam potom, pls, aj komentár, že je to kvôli tomu